include/mgp.hpp

```diff
@@ -3446,11 +3458,28 @@ void AddProcedure(mgp_proc_cb callback, std::string_view name, ProcedureType pro
   }
 
   for (const auto return_ : returns) {
-    auto return_name = return_.name.data();
-
+    const auto *return_name = return_.name.data();
     mgp::proc_add_result(proc, return_name, return_.GetMGPType());
   }
 }
+}  // namespace detail
+
+void AddProcedure(mgp_proc_cb callback, const std::string_view &name, ProcedureType proc_type,
+                  std::vector<Parameter> parameters, std::vector<Return> returns, mgp_module *module,
```

Comment:
check if it makes sense to add const + & to vectors in `AddProcedure` + `AddFunction`, I think Ante added that but just for consistency.

Reply:
I will actually put it back to just string_view for consistency. If I change it now to `const &` it might be breaking change and I think this is not a good idea. So I will leave it as it is.